awslabs/gluonts
# Adding torch.deepvar

Adding PyTorch implementation of DeepVAR (Vec-LSTM)

Modules added:
1. deepvar.estimator and deepvar.module
2. test_deepvar
3. distribution.lowrank_multivariate_normal (LowRankMultivariateNormalOutput)
4. fixed a bug in torch.deepar.lightning_module

Due to time constraint on my side (internship ending), I left the following additional features as TODO:
1. Use the `target_dimension_indicator` feature in training and prediction
2. Use Gaussian copula

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.


**Please tag this pr with at least one of these labels to make our release process faster:** BREAKING, new feature, bug fix, other change, dev setup
